plinic/src/components/pagination/Pagination.jsx

```diff
@@ -0,0 +1,85 @@
+import PaginationLib from 'react-js-pagination';
+import styled from 'styled-components';
+
+function Pagination({ activePage, totalItemsCount, handlePageChange }) {
+  return (
+    <PaginationBox>
+      <PaginationLib
+        activePage={activePage}
+        itemsCountPerPage={10}
+        totalItemsCount={totalItemsCount}
+        pageRangeDisplayed={5}
+        onChange={handlePageChange}
+        prevPageText="〈"
+        nextPageText="〉 "
+        hideFirstLastPages={true}
+      />
+    </PaginationBox>
+  );
+}
+
+export default Pagination;
+
+const Theme = {
+  navy: ({ theme }) => theme.color.navy,
+  white: ({ theme }) => theme.color.white,
+  gray: ({ theme }) => theme.color.gray,
+  font: [({ theme }) => theme.font.size['14'], ({ theme }) => theme.font.weight['normal']],
+  boldfont: ({ theme }) => theme.font.weight['bold'],
+  center: ({ theme }) => theme.align.flexCenter,
+  centerCol: ({ theme }) => theme.align.flexCenterColumn,
+};
```

Comment:
객체로 만들어서 사용한 방식 너무 좋습니다! 다만 각 키들을 상수로 사용하는 건 어떨까요?
아래는 예시입니다.
앗 그리고 객체를 변수에 넣어서 사용하는 것이므로 컴포넌트가 아니기 때문에 첫글자를 대문자로 쓰지 않아도 됩니다. 아니면, 혹시 theme를 사용하면 value에서 theme를 사용하기 때문에 에러가 나던가요?
```suggestion
const theme = {
  NAVY: ({ theme }) => theme.color.navy,
  WHITE: ({ theme }) => theme.color.white,
  GRAY: ({ theme }) => theme.color.gray,
  FONT: [({ theme }) => theme.font.size['14'], ({ theme }) => theme.font.weight['normal']],
  BOLD: ({ theme }) => theme.font.weight['bold'],
  CENTER: ({ theme }) => theme.align.flexCenter,
  CENTER_COLUMN: ({ theme }) => theme.align.flexCenterColumn,
};